src/auth/AuthClient.ts

```diff
@@ -271,9 +276,19 @@ export class AuthClient {
       emailNotifications || this.user.emailNotifications;
   }
 
-  async uploadProfileImage(image: any) {
-    await this.apiClient.uploadUserProfileImage(
-      image,
+  async uploadProfileImage(image: File): Promise<void>;
+  async uploadProfileImage(image: UploadableFile): Promise<void>;
+  async uploadProfileImage(image: UploadableFile | File) {
```

Comment:
I'm not sure if I understand these overloads. Why is `async uploadProfileImage(image: UploadableFile | File)` not enough?